crates/extricrate/src/lib.rs

```diff
@@ -62,20 +297,141 @@ pub mod dependencies {
 
     #[cfg(test)]
     mod tests {
+        use std::path::Path;
+
         use pretty_assertions::assert_eq;
+        use proc_macro2::LineColumn;
+        use syn::visit::Visit;
+
+        use crate::dependencies::{
+            File, ModuleName, NormalizedUseStatement, UseStatementType, UseVisitor,
+            list_use_statements,
+        };
+
+        #[test]
+        fn gets_a_simple_dependency() {
+            let test_project = Path::new(env!("CARGO_MANIFEST_DIR")).join("tests/fixtures/simple/");
+            let res = list_use_statements(&test_project).expect("Failed to list statements");
+
+            let main_statement = &res.get(&File("src/main.rs".to_owned())).unwrap()[0];
+            let module_a_statement = &res.get(&File("src/module_a/mod.rs".to_owned())).unwrap()[0];
+            assert_eq!(main_statement.source_module, "main".into());
+            assert_eq!(main_statement.target_modules, vec!["crate".into()]);
+            assert_eq!(
+                main_statement.statement.extent.0.start(),
+                LineColumn { line: 1, column: 0 }
+            );
+            assert_eq!(
+                main_statement.statement.extent.0.end(),
+                LineColumn {
+                    line: 1,
+                    column: 20
+                }
+            );
+            assert_eq!(
```

Comment:
Rather than test the item 0 (which might not exist), compare all the items with a Vec of one item